### DISCUSSION TITLE
Native AOT and CI/CD support (via Github Actions for example)

### DISCUSSION THREAD

**Discussion by alex-jitbit** [dotnet/runtime#95902] [Q&A] [answered]
The "prerequisites" section at  https://aka.ms/nativeaot-prerequisites mentions that I need a VS 2022 installation with "Desktop C++" component fully included for `PublishAOT` to work.

So, in other words, if I need to run AOT-publishing inside a Github Aciton or another CD tool, **I will need a full blown VS 2022 installation in the runner, correct?**

Or maybe a smaller setup is available? Like MSVC CLI-tools etc.

**Comment by hez2010** [comment, marked as answer]
What you only need is the MSVC build tools. 
There're prebuilt winserver images for CI which already contain such tools, alternatively you can also use the command line to deploy Visual Studio BuildTools 2022 during the CI.
